prep-javascript-objects/main.js

```diff
@@ -0,0 +1,17 @@
+const person = {
+  firstName: 'Michael',
+  lastName: 'Wood',
+  hobby: 'playing baseball'
+};
+console.log(person);
+
+person.fullName = 'Michael Wood';
+console.log("The person's name is:", person.fullName);
```

Comment:
We can constructor the person's full name by getting the values of their `firstName` and `lastName` properties.

```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
![prep-javascript-objects](https://user-images.githubusercontent.com/113574704/192588361-6f706665-10b9-47d1-b924-ca90808b1c4b.PNG)